packages/react-storage/src/components/StorageImage/__tests__/StorageImage.test.tsx

```diff
@@ -48,41 +55,176 @@ describe('StorageImage', () => {
     expect(img).toHaveClass(className);
   });
 
-  it('should get the presigned URL and pass it to image src attribute', async () => {
-    const onStorageError = jest.fn();
-    render(
+  it('should throw an error if both `imgKey` and `path` props are passed in', () => {
+    errorSpy.mockImplementation(() => {});
+
+    expect(() =>
+      render(
+        // @ts-expect-error testing invalid input
+        <StorageImage
+          alt="StorageImage"
+          path={path}
+          imgKey={imgKey}
+          onGetUrlError={onError}
+        />
+      )
+    ).toThrow('StorageImage cannot have both imgKey and path props.');
+
+    errorSpy.mockClear();
```

Comment:
Please move to the beforeEach

Reply:
Why would we want to run it before each test if it's only being used in one?